Makefile

```diff
@@ -13,18 +13,18 @@ wasm:
 	GOOS=js GOARCH=wasm ${BUILD_CMD} -ldflags "${LD_FLAGS}" -o ovhcloud.wasm ./cmd/ovhcloud
 
 test:
-	go test -v ./...
+	go test -v ./... ;
```

Comment:
I don't get this change, this is a shell line so adding a `;` should not change anything. Could you explain the change in behavior you expect ?

Reply:
Honestly this seems like an unconventional setup to have subdirectories in directories that you reference in $PATH (even the SO you linked says `You should check your PATH and make sure to remove any directories that contain such subdirectories` as first solution).

We would prefer to avoid running a shell just to prevent this issue.